gtk/transmission-gtk.appdata.xml.in

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2014 Richard Hughes <richard@hughsie.com>
+Copyright 2017 Endless Mobile, Inc.
+ -->
+<component type="desktop-application">
+  <id type="desktop">transmission-gtk.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0 OR GPL-3.0</project_license>
+
+  <_name>Transmission</_name>
+  <_summary>Download and share files over BitTorrent</_summary>
+
+  <description>
+    <!-- Translators: these are the application description paragraphs in the AppData file. -->
+    <_p>
+      BitTorrent is a peer-to-peer file-sharing protocol that is commonly used to
+      distribute large amounts of data between multiple users.
+    </_p>
+    <_p>
+      Transmission is a BitTorrent client with an easy-to-use frontend on top a
+      cross-platform backend.
+      Native frontends are available for OS X and Windows, as well as command line and
+      web frontends.
+    </_p>
+    <_p>
+      Notable features of Transmission include Local Peer Discovery and Full Encryption,
+      Full encryption, DHTµTP, PEX and Magnet Link support.
+    </_p>
+  </description>
+  <url type="homepage">https://www.transmissionbt.com/</url>
+  <screenshots>
+    <screenshot type="default">
+      <!-- Translators: This should be the URI of a screenshot of Transmission-Gtk in your language. If you can't take a screenshot, leave the string as the English screenshot. -->
+      <_image type="source">https://raw.githubusercontent.com/transmission/transmission/master/gtk/screenshots/a.png</_image>
```

Comment:
I was about to claim that this is pretty standard in appdata files – wouldn't users like to see screenshots in their own language? As a native English speaker this is not a situation I personally encounter – but I have checked several GNOME applications (Dictionary, Builder, Clocks) and none of them translate screenshot URLs, so you and they have convinced me.